UPGRADE.md

```diff
@@ -0,0 +1,14 @@
+Upgrade from 0.x to 1.x
```

Comment:
this contradicts with 

```
Integration of JSON API with Symfony which supports JMS Serializer 2.0. 

 Note! Old version of JMS Serializer 1.0 support will end soon. 
Please use JsonApiBundle version [1.0](https://github.com/ecentria/JsonApiBundle/tree/1.0/) instead.
```

Also this file should be called CHANGELOG, but UPGRADE should have a brief guide of upgrade